Gems/Multiplayer/Code/Include/Multiplayer/Components/MultiplayerComponentRegistry.h

```diff
@@ -67,11 +67,26 @@ namespace Multiplayer
         //! @return reference to the requested component data, an empty container will be returned if the NetComponentId does not exist
         const ComponentData& GetMultiplayerComponentData(NetComponentId netComponentId) const;
 
+        //! Returns the combined hashes of all the multiplayer components creating a single holistic version hash that can be compared between the server and client app.
+        //! @return a 64-bit hash value representing of all of the multiplayer components
+        AZ::HashValue64 GetMultiplayerComponentVersionHolisticHash() const;
```

Comment:
Holistic seems a bit weird too 😆 CombinedHash or AggregateHash might read a bit better? Perhaps even ComponentRegistryHash, although that is definitely a bit more esoteric..

Reply:
Maybe just GetMultiplayerSystemVersionHash? I think the focus here should be on describing what it is not how it's formed.

Reply:
I like it! Name changed to SystemHash.